doc/man7/fips_module.pod

```diff
@@ -88,6 +89,9 @@ Edit the config file to add the following lines near the beginning:
     [base_sect]
     activate = 1
 
+    [algorithm_sect]
+    default_properties = fips=yes
+
```

Comment:
Hmm, should also the various fips configs in `tests/` be amended?

Reply:
I don't see a need.  Of the four relevant configuration files:
| File | Notes |
| --- | --- |
| `default-and-fips.cnf` | Having `fips=yes` is inappropriate here |
| `fips-alt.cnf` | Does via the `fips_mode` setting |
| `fips-and-base.cnf` | Doesn't have `fips=yes` |
| `fips.cnf` | Has `fips=yes` already |

We could add the property to the `fips-and-base.cnf` file, but we'd lose the only FIPS specific test case where we don't set the property query (and we really ought to have such a test).